examples/gcoap/gcoap_cli.c

```diff
@@ -88,16 +91,50 @@ static void _resp_handler(unsigned req_state, coap_pkt_t* pdu,
 }
 
 /*
- * Server callback for /cli/stats. Returns the count of packets sent by the
- * CLI.
+ * Server callback for /cli/stats. Accepts either a GET or a PUT.
+ *
+ * GET: Returns the count of packets sent by the CLI.
+ * PUT: Updates the count of packets. Rejects an obviously bad request, but
+ *      allows any two byte value for example purposes. Semantically, the only
+ *      valid action is to set the value to 0.
  */
 static ssize_t _stats_handler(coap_pkt_t* pdu, uint8_t *buf, size_t len)
 {
-    gcoap_resp_init(pdu, buf, len, COAP_CODE_CONTENT);
+    /* read coap method type in packet */
+    unsigned method_flag = coap_method2flag(coap_get_code_detail(pdu));
+
+    switch(method_flag) {
+        case COAP_GET:
```

Comment:
Minor issue: cases should be at same indent level as switch.

Reply:
well, looking at other places in the code, this seems to be the more common style.

Reply:
I'm trying to do better myself [1], and comply with the coding conventions. :-) I don't feel strongly about this issue. 

[1] https://github.com/RIOT-OS/RIOT/pull/7337/files#diff-1ad46e1aa545fa54a91a5563e03a4ee8